docs/events/timestamps/index.md

```diff
@@ -0,0 +1,39 @@
+| `true_tstamp`         | You                                                                                 | Exact timestamp, defined within your tracking code                                                                                                                     | ❌             |
+| `collector_tstamp`    | [Collector](/docs/pipeline/collector/index.md)                                      | When the Collector received the event payload                                                                                                                          | ✅             |
+| `derived_tstamp`      | [Enrich](/docs/api-reference/enrichment-components/index.md)                        | Calculated from other timestamps, or the same as `true_tstamp`                                                                                                         | ✅             |
+| `etl_tstamp`          | [Enrich](/docs/api-reference/enrichment-components/index.md)                        | The time at which Enrich started processing the event                                                                                                                  | ✅             |
+| `refr_device_tstamp`  | [Enrich](/docs/api-reference/enrichment-components/index.md)                        | Timestamp extracted from the [cross-domain navigation query string](/docs/pipeline/enrichments/available-enrichments/cross-navigation-enrichment/index.md), if present | ❌             |
+| `load_tstamp`         | [Loader](/docs/destinations/warehouses-lakes/loading-process/index.md) or warehouse | Timestamp when the event was loaded into the warehouse                                                                                                                 | ✅             |
+
+The `load_tstamp` is added either by the Loader or by the warehouse at the point of loading, depending on the Loader/warehouse. Use this timestamp for incremental processing.
+
+## Derived timestamp
+
+The `derived_tstamp` corrects for inaccurate timestamps and delays.
+
+It's calculated as `collector_tstamp` minus the difference between the `dvce_sent_tstamp` and the `dvce_created_tstamp`. Alternatively, if `true_tstamp` is available, `derived_tstamp` is the same as `true_tstamp`.
+
+The calculation has two assumptions:
+* We assume that, although `dvce_created_tstamp` and `dvce_sent_tstamp` can both be inaccurate, they're inaccurate in precisely the same way. If the device clock is 15 minutes fast at event creation, then it remains 15 minutes fast at event sending, whenever that might be.
+* We assume that the time taken for an event to get from the device to the Collector is neglible, i.e. the lag between `dvce_sent_tstamp` and `collector_tstamp` is 0 seconds.
+
+## What timestamp to use?
```

Comment:
I would call it out that it depends on how you plan on using the data. If you want to use it to filter on the `atomic.events` table directly then for sure they would need to use the timestamp it is partitioned on. For actual reporting, the `derived_tstamp` is advised to be used throughout (this is what we default as `start_tstamp` in our packaged modelled derived tables). The rest are for custom use cases or for debugging (?) to understand more on how the data is formed/sent/integrated into our pipeline/their warehouse etc.